### PR TITLE
apps/ltr308: add simple application to test ltr308 sensor

### PR DESCRIPTION
**Pull request by robertalexa2000**
Signed-off-by: Robert-Ionut Alexa <robertalexa2000@gmail.com>

## Summary
I have already opened a pull request for adding support for the LTR308 sensor. This commits provides a sample application for testing it.

### CODE REVIEW

File: examples/ltr308/ltr308_main.c [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+  ret = ioctl(fd, SNIOC_GET_DATA, &data);
+  if (ret < 0)
+    {
+      perror("Failed to get data");
+      return ret;
+    }
+
+  printf("Data: %x\n", data);
```
Comment (pkarashchenko) [inline examples/ltr308/ltr308_main.c:70]: Close fd

File: examples/ltr308/ltr308_main.c [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      return fd;
+    }
+
+  ctrl.reset = 0;
+  ctrl.enable = 1;
+  ret = ioctl(fd, SNIOC_SET_CTRL, &ctrl);
+  if (ret < 0)
+    {
+      perror("Failed to enable the sensors");
+      return ret;
```
Comment (pkarashchenko) [inline examples/ltr308/ltr308_main.c:60]: Close fd

File: examples/ltr308/ltr308_main.c [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+  ret = ioctl(fd, SNIOC_GET_DATA, &data);
+  if (ret < 0)
+    {
+      perror("Failed to get data");
+      return ret;
```
Comment (pkarashchenko) [inline examples/ltr308/ltr308_main.c:67]: Close fd

**Comment by Donny9** [comment]
@robertalexa2000 You can use uorb_listener tools to monitor light value if the driver of light register by sensor_register(driver/sensors/sensor.c)

**Comment by xiaoxiang781216** [comment]
@robertalexa2000 do we still need this sample program after https://github.com/apache/incubator-nuttx/pull/7466 utilize the sensor framework?

**Comment by robertalexa2000** [comment]
@xiaoxiang781216 I will update this with a new example that I used for testing and you can decide if we merge it or not. I think it might be useful to exemplify how to use the .calibrate callback

**Comment by xiaoxiang781216** [comment]
> @xiaoxiang781216 I will update this with a new example that I used for testing and you can decide if we merge it or not. I think it might be useful to exemplify how to use the .calibrate callback

Ok, let's merge your new patch.

**Comment by robertalexa2000** [comment]
I'm closing this request and will open a new one later.

**Pull request closed without merging**